### ISSUE TITLE
A11y: Learn.wordpress.org / learning pathways links

### ISSUE THREAD

**Issue by joedolson** [WordPress/Learn#2770]
## Type of feedback
//dev

## Description

The links for learning pathways do not look like links, and have insufficient hover state.

First, I only realized these were links when I was doing testing with a screen reader; prior to doing the screen reader testing, I had simply scanned past them assuming they were ornamental content. This suggests a design problem that might need some clarification. 

Second, the hover state is a color-only change to the background from `#fff` to `#f6f6f6`, which is a nominal contrast difference. For low vision or in bright ambient light, particularly with low contrast vision, this change may not be noticeable. In itself, not an accessibility violation, because of the cursor change, but worth noting given how hard it is to recognize that these are links in the first place.

This is a very minor issue, overall, but I think it could be improved.

## Screenshots or screen recording (optional)

![image](https://github.com/user-attachments/assets/dcd67edb-2862-4670-85f1-f92ff44252bb)

**Comment by adamwoodnz** [comment]
@WordPress/meta-design your input please

**Comment by marko-srb** [comment]
Can we simply underline the title on hover (together with the background gray change - it should stay) and have it solved like that? I think visually that works well. We can always reconsider it after the launch if we want some different treatment.

**Comment by joedolson** [comment]
That would definitely make the interactive state more obvious; I still don't feel that there's a lot of affordance to tell me that this is a link in the first place.

**Comment by marko-srb** [comment]
@joedolson 

I am not sure I can fully follow the issue at hand. Firstly I do recognize this isn't looking like a link, but more like a notification or banner to read (on some very abstract level). Yet, this does look like a card that could have some interaction — other sites have cards that only provide shadow when hovered — meaning: if we underline the title + already present cursor change, I think we solved the problem entirely. 

I am now trying to guess: you might want to see some more drastic change (like blue color - white text on hover, etc) but that won't work due to the style of the page visually. It will not be appropriate. Then, there's another guess: we might zoom in the illustration when hovered — but again this doesn't fit to anything we are doing with hover states, so it's a no go for me. 

—

Let me know why underline + cursor switch + light gray switch doesn't do it for you?

**Comment by ryelle** [comment]
This sounds like the same problem we talked about on [the main site homepage (latest news section)](https://github.com/WordPress/wporg-main-2022/issues/472#issuecomment-2239696057), where the black headings don't look like links.

> Let me know why underline + cursor switch + light gray switch doesn't do it for you?

As I understand it, those things only appear once you've hovered/interacted with the content, but with just looking at the page, you wouldn't know those were links.

What if the headings were blue to match the other links on this page? The hover could stay the same. This matches how we've used cards on other sites, like [Developer Resources](https://developer.wordpress.org/).

<img width="1244" alt="Screenshot 2024-07-26 at 2 05 39 PM" src="https://github.com/user-attachments/assets/af4875ac-625a-494d-b05a-9b291a7f7724">

**Comment by joedolson** [comment]
Yes, that's what I was calling out; the proposed hover state is great, but the problem is in the static state. 

I think that going to blue would be sufficient, and matches many other patterns used elsewhere in the redesign, which helps with learnability, as well.

**Comment by marko-srb** [comment]
Yes, switching to blue works well!

**Comment by fcoveram** [comment]
Agree with switching the title to blue. Thanks for raising this.
